styles/facts.css

```diff
@@ -0,0 +1,68 @@
+.fact-box {
+    width: 20rem;
+    padding: 20px;
+    border-radius: 24px;
+    border-style: solid;
+    border-width: 2px;
+    border-color: rgba(101, 81, 44, 0.3);
+    background-color: #fffbee;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    text-align: center;
+    margin: 20px;
+}
+
+.fact-image {
+    width: 15rem;
+    border-radius: 10px;
+    margin-bottom: 15px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    text-align: center;
+    float:left;
+}
+
+/*background-color: ;*/
+/*padding: 20px;*/
+/*margin: 20px;*/
+/*border-radius: 10px;*/
```

Comment:
Like source files in Python, we want to clean up commented or tester code once we have everything working.